app/src/main/java/org/simple/clinic/drugs/search/Drug.kt

```diff
@@ -48,5 +49,11 @@ data class Drug(
 
     @Query("SELECT COUNT(uuid) FROM Protocol")
     fun count(): Observable<Int>
+
+    @Query("""
+      SELECT drugs.*, INSTR(lower(drugs.name), lower(:query)) namePosition FROM Drug drugs
+      WHERE namePosition > 0
```

Comment:
We are already using `INSTR` in the project and that seems to be the latest implementation in sqlite. So I think it makes sense to use this 🤔

Reply:
The only reason we used that was because we wanted the number position of the search query to contribute to the sort ordering. Here, we don't need to do that so we can use the `LIKE` clause which is simpler.

What do you think?

Reply:
Makes sense. Making that change in this PR